suite/sanity/failover.go

```diff
@@ -0,0 +1,37 @@
+package sanity
+
+import (
+	"github.com/gravitational/robotest/infra/gravity"
+)
+
+// TODO
+type failoverParam struct {
+	installParam
+}
+
+// failover tests master failover
+func failover(p interface{}) (gravity.TestFunc, error) {
```

Comment:
why is this an interface{} instead of `failoverParam`?

Reply:
Implements ConfigFn
`// ConfigFn is a function which takes default parameters and returns a pre-configured test function
 type ConfigFn func(param interface{}) (gravity.TestFunc, error)`